languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml

```diff
@@ -11944,7 +11944,7 @@ USA
     <category id='SHORTEN_IT' name="Linguagem concisa" type="style">
 
 
-        <rule id='DAR_VALOR_VALORIZAR' name="Simplificar: Verbo 'dar' + valor → valorizar" default="temp_off"> 
+        <rule id='DAR_VALOR_VALORIZAR' name="Simplificar: Verbo 'dar' + valor → valorizar"> 
```

Comment:
Can we make this `picky` too? Since `shorten_it` is not active yet, this will show up for every writing goal/won't be filtered as style. Perhaps the tone tag `objective` could be a temporary alternative.